tests/Features/Expect/each.php

```diff
@@ -79,11 +79,13 @@
 });
 
 it('accepts callables', function () {
```

Comment:
Can you add a new test instead of modifying an existing one? For testing keys.

Reply:
All done :)